redash/models/__init__.py

```diff
@@ -772,6 +773,10 @@ def evaluate(self):
     def subscribers(self):
         return User.query.join(AlertSubscription).filter(AlertSubscription.alert == self)
 
+    def render_template(self, showError=None):
```

Comment:
`showError` seems to be unused? Also should be `show_error`.

Reply:
thanks! I've removed.